lib/membrane_opus/description.ex

```diff
@@ -1,42 +1,37 @@
-defmodule Membrane.Caps.Audio.Opus do
+defmodule Membrane.Opus.Description do
   @moduledoc """
-  This module implements struct for caps representing a single Opus-encoded
-  audio frame.
+  This module implements struct describing a single Opus-encoded audio frame.
   """
 
   # Opus frame duration type.
   #
   # 2 means 2.5ms, other values just map into duration in miliseconds.
   @type frame_duration_t :: 2 | 5 | 10 | 20 | 40 | 60
```

Comment:
Also, the RFC mentions combining frames into packet of 120ms (https://tools.ietf.org/html/rfc6716#page-10) Not sure if we want to implement that now, but maybe some TODO or NOTE should be put here